.github/workflows/main.yml

```diff
@@ -0,0 +1,28 @@
+name: Continuous integration
+
+on: [pull_request]
+
+jobs:
+  cypress-run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
```

Comment:
as in the nvmrc you are specifying version v19.3, you should use same version here

Reply:
updated version according to the comment.